source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,286 @@
+=========================================
+
+You can create an account with any of the officially provided wallets from Concordium. This tutorial focuses on the mobile wallet but feel free to explore the setup of the :ref:`desktop wallets<overview-account-desktop>` alternatively.
+
+You can download the mobile wallet package from the :ref:`installation page<downloads-mobile-wallet-testnet>`.
+If you follow the instruction in the mobile app you are asked to name your account and to name your initial identity card. This tutorial uses `Concordium` for the account name and `Account1` for the identity card name but feel free to chose your own naming.
+
+
+.. image:: ./images/pb_tutorial_1.png
+   :width: 20 %
+
+.. image:: ./images/pb_tutorial_2.png
+   :width: 20 %
+
+.. image:: ./images/pb_tutorial_3.png
+   :width: 20 %
+
+
+You don't have to provide an ID to create an account on testnet when selecting `Concordium testnet IP`. This allows you to insert some dummy identity data and is meant for testnet testing only.
+
+.. image:: ./images/pb_tutorial_4.png
+   :width: 20 %
+
+.. Note::
+   On mainnet you will have to provide your personal ID to an identity provider to get a mainnet account.
+
+You also have to request some testnet CCD. The mobile app wallet has a button that you can use to request 20000 testnet CCD.
+
+.. image:: ./images/pb_tutorial_5.png
+   :width: 20.5 %
+.. image:: ./images/pb_tutorial_6.png
+   :width: 20 %
+
+.. Note::
+   Some CCD on your testnet account is needed later when sending transactions from your account to the testnet blockchain.
+
+Create a backup of your wallet by clicking the `Backup` button in the mobile app. Safe the file `concordium-backup.concordiumwallet` for now because it will be used in the next section.
+
+.. image:: ./images/pb_tutorial_7.png
+   :width: 20 %
+
+.. Note::
+   Please remember the `account name` and the `export password` that you used for creating the backup file. These two variables are needed later when important your account key into the testnet node.
+
+
+Import your mobile wallet account key to your testnet node
+==========================================================
+
+You are ready now to import your keys to your testnet node. You have to transfer your wallet backup file (meaning the file `concordium-backup.concordiumwallet`) to the place where your testnet node is running at.
+
+
+.. Note::
+   You can use a file-sharing tool (such as `FileZilla`) to transfer your wallet backup file from e.g. your local machine to your instance.
+
+If you used the desktop wallet for creating your account, you can find the equivalent steps :ref:`here<export-import>` for exporting and backing up your wallet account.
+
+You are set for importing your key to your testnet node now. Remember that you created an account with name `Concordium` earlier in this tutorial:
+
+.. code-block:: console
+
+   $./concordium-client config account import ./concordium-backup.concordiumwallet --name Concordium
```

Comment:
It actually uses the name(s) specified in the app by default, so there is no need to use the `--name` flag.
So the users don't need to remember the account name, as you described further up.

Reply:
I removed the flag and the comment about remembering it.